### DISCUSSION TITLE
Define the state that a state_color turns an icon yellow.

### DISCUSSION THREAD

**Discussion by gregg098** [home-assistant/frontend#11283] [Other feature requests]
## The request
On my main Lovelace screen, I have a horizontal stack with three button cards. Each button is a person entity. The tap action navigates to the map screen.  Everything works as expected, except that if I change "state_color" to true, nothing changes for these entities. It would be nice to have it turn yellow (like when a switch is on) when "home" or maybe the ability to define what state that a state_color activates on for entities that aren't just switches/binary.

This applies to all card types that support state_color

## The alternatives
I have looked into some template sensors with template icons and things like that, and it works OK, but it would still be a nice feature to just work out of the box.

**Comment by zsarnett** [comment]
Check out your issue... You placed your text in the comments `<-- --!>` Anything in those will not be shown

**Comment by gregg098** [comment]
> Check out your issue... You placed your text in the comments `<-- --!>` Anything in those will not be shown

Fixed. Thanks.

**Comment by broyuken** [comment]
I am experiencing a similar issue with device trackers. I would like to have entities that are home be one color, and entities that are not home another color in a glance card. I use the netgear wifi tracker to track if certain devices are on my network and would like to be able to see at a "glance" which devices are offline. This used to work until I updated to 105

**Comment by broyuken** [comment]
Anything we can do to get these to display correctly again?
